packages/cli/cli/build.prod.cjs

```diff
@@ -9,7 +9,7 @@ async function main() {
     await tsup.build({
         entry: ['src/cli.ts'],
         format: ['cjs'],
-        minify: true,
+        minify: false,
```

Comment:
Happy to revert this, it adds an extra 30mb to the CLI (~30mb -> ~60mb). But I imagine it would make understanding user's error messages a lot easier. (e.g. things like this: https://github.com/fern-api/fern/pull/4675)